### PR TITLE
Feature/military law changes

### PR DESCRIPTION
**Pull request by x3igh7**
* changes to military laws to make them more interesting choices

### CODE REVIEW

File: common/laws/00_monarchy.txt [hunk 1]
```diff
@@ -87,9 +93,14 @@ monarchy_military_reforms = {
 	
 	mercenary_reliance = {
 		modifier = {
-			mercenary_land_maintenance_cost = -0.15
-			recruit_mercenary_cost_modifier = -0.30
+			mercenary_land_maintenance_cost = -0.20
+			recruit_mercenary_cost_modifier = -0.35
 			max_mercenary_stacks = 1
+			levy_size_multiplier = -0.05
```
Comment (DaFoogle) [inline common/laws/00_monarchy.txt:99]: i feel this should be larger if a nations player chooses to use more mercs.

Comment (DaFoogle) [reply common/laws/00_monarchy.txt:99]: but only by a small margin perhaps -7.5% or -7%

**Comment by SnowletTV** [comment]
Apparently you need to comment to approve

**Review by SnowletTV — APPROVED** [review]
Remember DaFoogle suggestion

**Pull request merged**